tests/test_code_gen.py

```diff
@@ -403,5 +403,12 @@ def test_annassign(self):
         """
         self.assertAstEqualIfAtLeastVersion(source, (3, 6))
 
+
+    def test_compile_types(self):
+        code = '(a + b + c) * (d + e + f)\n'
+        for mode in 'exec eval single'.split():
+            srcast = compile(code, 'dummy', mode, ast.PyCF_ONLY_AST)
+            dsttxt = astor.to_source(srcast)
```

Comment:
There should be an assert to test that `code` equals to `dsttxt` (or something similar), right?